### ISSUE TITLE
Members was not added to all spaces via superinvites

### ISSUE THREAD

**Issue by emilvincentz** [acterglobal/a3#1627]
this is also the case for spaces.In this case, i had made a superinvite that should have added the member to the space. 
For some of the spaces added to the invite code, he joined, but others he didnt joined. invite code: gpkenya04

**Comment by gnunicorn** [comment]
@emilvincentz do you know _who_'s account that was? would help with debugging this...

**Comment by emilvincentz** [comment]
Yes, it was for Stephen Okwany
@okwany:acter.global

**Comment by gnunicorn** [comment]
The server has no record of that user accepting that invite. That could be because adding them to some room in that list had failed and thus the other rooms have been skipped.

This change https://github.com/acterglobal/synapse-super-invites/commit/97f545ace0750c2579ce3beb91ab4fe66b4dd675 in the server side feature fixes that problem by continuing even if the user can't be added to any number of rooms they are supposed to be added to. It marks them the errors in the database so we can investigate it, too. this should prevent this going forward.

**Comment by emilvincentz** [comment]
Reopening this issue, as this also seems to happen with PxP onboarding

**Comment by emilvincentz** [comment]
okey, i think what is happening is that some people get addded to the spaces and chats as they should, while the remaining shows up as a requests in the acitivty screen

**Comment by gnunicorn** [comment]
> while the remaining shows up as a requests in the acitivty screen

Curious ...

@emilvincentz how many rooms are there in total in the super invite?

**Comment by kumarpalsinh25** [comment]
@emilvincentz     
Is this issue is still persists? Can you check and confirm?

**Comment by emilvincentz** [comment]
Yes still persists, but i think @gnunicorn is working on fix for this?

**Comment by gnunicorn** [comment]
fixed by #2552 .
